novoda-blog-diversity/lib/plugin.js

```diff
@@ -8,7 +8,7 @@ const component = {
 
 const configuration = () => {
     return {
-        "blog-diversity-dashboard-plugin": {
+        novoda_blog_diversity: {
```

Comment:
changing this `id` means we'll need to delete the plugin from the admin and readd. Using `""` to declare the key is also valid but I suppose it's better for consistency if we don't